src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -284,7 +285,9 @@ def step(
 
         # 3. Clip "predicted x_0"
         if self.config.clip_sample:
-            pred_original_sample = torch.clamp(pred_original_sample, -1, 1)
+            pred_original_sample = torch.clamp(
+                pred_original_sample, -self.config.clip_sample_range, self.config.clip_sample_range
+            )
 
```

Comment:
Required for using the prior with DDPMScheduler. This is backwards compatible because of the default config value of clip_sample_range

Reply:
Perfect!